### PR TITLE
H-02 Image layout homework

### PR DESCRIPTION
**Pull request by elegorod**
02-image-layout - actual hometask
02-image-layout-draft1 folder - simple draft

### CODE REVIEW

File: homeworks/02-image-layout-draft1/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This is first draft, actual solution is located in `02-image-layout` folder.
+
+Homework to make image layout.
+
+# Demo
+TODO
```
Comment (manapard) [inline homeworks/02-image-layout-draft1/README.md]: Please, deploy and provide a link to the demo app :)

File: homeworks/02-image-layout/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    for (let i = 1; i <= 35; i++) {
+      const image = new Image()
+      image.onload = this.imageLoaded.bind(this)
+      image.src = require(`./images/${i}.jpg`).default
+      this.images.push(image)
+    }
+  }
+
+  imageLoaded() {
+    this.setState(state => ({loadedCount: state.loadedCount + 1}))
```
Comment (manapard) [inline homeworks/02-image-layout/src/App.tsx:28]: I don't see, where do you use this counter in the render function?

File: homeworks/02-image-layout-draft1/main.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.imageContainer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 3px;
+}
+
+.imageContainer img {
+  height: 220px;
```
Comment (manapard) [inline homeworks/02-image-layout-draft1/main.css:9]: this would be too easy :wink:

File: homeworks/02-image-layout/src/App.tsx (hunk 2 above)
Comment (elegorod) [reply homeworks/02-image-layout/src/App.tsx:28]: It's not used, it's just a way to force re-render.

File: homeworks/02-image-layout-draft1/main.css (hunk 3 above)
Comment (elegorod) [reply homeworks/02-image-layout-draft1/main.css:9]: That's why it's draft. Hard way is in 02-image-layout folder.

File: homeworks/02-image-layout-draft1/README.md (hunk 1 above)
Comment (elegorod) [reply homeworks/02-image-layout-draft1/README.md]: Done, so there are 2 demos for this homework now.

**Pull request merged**